src/main/flight/pid.c

```diff
@@ -1124,6 +1150,22 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
         }
 #endif
         // calculating the PID sum
+
+        // P boost at the end of throttle chop
+        // attenuate effect if turning more than 50 deg/s, half at 100 deg/s
+        float agBoostAttenuator = fabsf(currentPidSetpoint) / 50.0f;
+        if (agBoostAttenuator < 1.0f) {
+            agBoostAttenuator = 1.0f;
+        }
+        const float agBoost = 1.0f + (pidRuntime.antiGravityPBoost / agBoostAttenuator);
+        pidData[axis].P *= agBoost;
+        if (axis == FD_ROLL){
```

Comment:
:space_invader:

Reply:
cryptic :-) - was the concern a potential divide by zero bug?
`agBoostAttenuator` can't go to zero... 
or was something else a cause for concern?

Reply:
add a white space `if (axis == FD_ROLL) {`

Reply:
To be read as 'some (white-) space (inconsistency) has invaded your code'. :-D